### ISSUE TITLE
Rename songs,albums, artists, change metadata , edit tags?

### ISSUE THREAD

**Issue by zoobporsor** [audacious-media-player/audacious#1450]
**Description**
Which feature or functionality are you missing in Audacious?
What makes it beneficial for users?

**Additional information**
* Do other audio players offer this feature?
* Screenshots

Is it possible to Rename songs/artists/albums edit the metadata and tags in Audacious? How?

**Comment by radioactiveman** [comment]
Yes, writing tags for WAV files is not supported. I leave this feature request open but have to say that it has no high priority for us. 
If someone wants to implement it in `libaudtag`, patches are of course welcome. :)

**Comment by atomGit** [comment]
just out of curiosity, why are you storing music in wave format? if you're unaware, flac is lossless and it supports metadata

if for some reason you don't want a compressed file (extra CPU cycles to decompress), run flac with `--compression-level-0`
